ktor-client/ktor-client-core/common/src/io/ktor/client/request/HttpRequest.kt

```diff
@@ -83,7 +83,7 @@ class HttpRequestBuilder : HttpMessageBuilder {
      * A deferred used to control the execution of this request.
      */
     @KtorExperimentalAPI
-    val executionContext: Job = Job()
+    var executionContext: Job? = null
```

Comment:
unsafe public API

Reply:
I made it `lateinit`.

Reply:
Changed on just `var` with replacement in `HttpClientEngine`.